script.js

```diff
@@ -138,18 +138,21 @@ contactFrom.addEventListener('submit', (event) => {
   }
 });
 
-const storeName = document.querySelector('#email');
-const storeEmail = document.querySelector('#name');
-const storeMesssage = document.querySelector('#message');
+const storeName = document.querySelector('email');
+const storeEmail = document.querySelector('name');
+const storeMesssage = document.querySelector('message');
```

Comment:
- Good job so far 👍 but you have not used the `querySelector` here very well and as a result, the information is not persisted in the browser.
- [ ] First, on lines 141 and 142, you have exchanged the `ids`, Instead of selecting the name on line 141 you are selecting the email and the opposite has been done on line 142. Do well to correct this.
- [ ] Next to select the `id`, a `#` is needed so you should have something like this: `const storeName = document.querySelector('#name');` Kindly rectify this issue 👍